doc/CHECKINS.md

```diff
@@ -32,13 +32,8 @@ notify everyone by referencing `@/all` and linking the post. Include a brief mes
 
   - Troubleshot any issues that the rest of the community can learn from?
 
-- Include sections from the last post that make sense to carry over - if you need additional guidance reference past check-ins.
+- Include sections from the [last check-in](https://github.com/publiclab/plots2/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+check-in%22) that make sense to carry over. If you need hint for check-in template, theme or content, refer [past check-ins](https://github.com/publiclab/plots2/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22community+check-in%22+).
```

Comment:
Just a small change - 
1) if you need _a_ hint, _reference_ the template, theme, or content of [past check-ins](https://github.com/publiclab/plots2/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22community+check-in%22+)

Reply:
OK. I am changing this line. :+1:

Reply:
Done.